conda/recipes/libraft/install_libraft_distance.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 # Copyright (c) 2022, NVIDIA CORPORATION.
 
-./build.sh libraft --install -v --allgpuarch --no-nvtx
+cmake --install cpp/build --component raft_distance
```

Comment:
@robertmaynard hoping to have your thoughts here. Should this be enough to install the proper cmake files along w/ the artifact?